generation/WinSDK/enums.json

```diff
@@ -11900,25 +11900,70 @@
     ]
   },
   {
+    "namespace": "Windows.Win32.System.SystemInformation",
```

Comment:
You shouldn't need to specify this explicitly. Just reorder the "uses" below so that SYSTEM_INFO is the first one and the enum should move to that namespace.

Reply:
I'm not sure I like that implicit "first usage defines the namespace" behavior over being explicit. But I guess the thinking here is that this enum should _follow_ around the `SYSTEM_INFO` struct. Will make that edit.